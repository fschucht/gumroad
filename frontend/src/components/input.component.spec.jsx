import renderer from 'react-test-renderer';
import { Input } from './input.component';

describe('Input', () => {
  it('should render a Input component', () => {
    const renderedComponent = renderer.create(<Input type='text' value='Test' placeholder='Test' />).toJSON()

    expect(renderedComponent).toMatchSnapshot()
  })
})