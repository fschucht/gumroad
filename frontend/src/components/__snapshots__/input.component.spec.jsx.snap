// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input should render a Input component 1`] = `
<input
  className="input"
  placeholder="Test"
  type="text"
  value="Test"
/>
`;
